modules/consensus/setdb.go

```diff
@@ -200,6 +205,9 @@ func (db *setDB) rmItem(bucket string, key interface{}) error {
 
 // inBucket checks if an item with the given key is in the bucket
 func (db *setDB) inBucket(bucket string, key interface{}) bool {
+	if key == nil {
+		return false
+	}
```

Comment:
Along similar lines, is this code ever supposed to be reached? If not, it should probably just be a `if DEBUG { panic() }`